src/Microsoft.ML.DataView/DataViewRowCursor.md

```diff
@@ -33,26 +33,28 @@ cursor until the `Batch` ID changes. Whereupon, the consolidator will find the
 next cursor with the next lowest batch ID (which should be greater, of course,
 than the `Batch` value we were just iterating on).
 
-Put another way: if we called `GetRowCursor` (possibly with an `IRandom`
+Put another way: if we called `GetRowCursor` (possibly with a `Random`
 instance), and we store all the values from the rows from that cursoring in
 some list, in order. Now, imagine we create `GetRowCursorSet` (with an
```

Comment:
>. [](start = 19, length = 1)

This sentence starts with "If" but doesn't have a "then".

Reply:
Ah thanks. If there is an "if" then there should also be a "then." I think I probably meant "suppose" not "if." So that's what I'll do.